compiler/src/iree/compiler/Codegen/SPIRV/test/link_executables.mlir

```diff
@@ -382,6 +422,10 @@ hal.executable private @dispatch_0 {
 // dispatch_0. So it can link with dispatch_0.
 hal.executable private @dispatch_1 {
   hal.executable.variant @spirv_0 target(#vulkan_target_0) {
+    hal.executable.condition(%arg0: !hal.device) -> i1 {
```

Comment:
these return true regions shouldn't be required - are they today? (we can fix that)

Reply:
Yeah. Removed and avoided generating such condition ops.